src/Sulu/Bundle/AdminBundle/Resources/js/components/DatePicker/tests/__snapshots__/DatePicker.test.js.snap

```diff
@@ -1,1109 +1,1270 @@
+"<div>
+  <div class=\\"container\\">
+    <div style=\\"top: 10px; left: 34px; position: fixed; pointer-events: auto;\\">
+      <div class=\\"rdt\\">
+        <div></div>
+        <div class=\\"rdtPicker\\">
+          <div class=\\"rdtYears\\">
+            <table>
+              <thead>
+                <tr>
+                  <th class=\\"rdtPrev\\"><span>‹</span></th>
+                  <th class=\\"rdtSwitch\\" colspan=\\"2\\">2010-2019</th>
+                  <th class=\\"rdtNext\\"><span>›</span></th>
+                </tr>
+              </thead>
+            </table>
+            <table>
+              <tbody>
+                <tr>
+                  <td data-value=\\"2009\\" class=\\"rdtYear\\">2009</td>
+                  <td data-value=\\"2010\\" class=\\"rdtYear\\">2010</td>
+                  <td data-value=\\"2011\\" class=\\"rdtYear\\">2011</td>
+                  <td data-value=\\"2012\\" class=\\"rdtYear\\">2012</td>
+                </tr>
+                <tr>
+                  <td data-value=\\"2013\\" class=\\"rdtYear\\">2013</td>
+                  <td data-value=\\"2014\\" class=\\"rdtYear\\">2014</td>
+                  <td data-value=\\"2015\\" class=\\"rdtYear\\">2015</td>
+                  <td data-value=\\"2016\\" class=\\"rdtYear\\">2016</td>
+                </tr>
+                <tr>
+                  <td data-value=\\"2017\\" class=\\"rdtYear\\">2017</td>
+                  <td data-value=\\"2018\\" class=\\"rdtYear\\">2018</td>
+                  <td data-value=\\"2019\\" class=\\"rdtYear\\">2019</td>
+                  <td data-value=\\"2020\\" class=\\"rdtYear\\">2020</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>"
+`;
+
 exports[`DatePicker should render date picker with time picker 1`] = `
 <div>
-  <div />
+  <div>
+    <label
+      class="input default left"
+    >
+      <div
+        class="prependedContainer default"
+      >
+        <span
+          aria-label="su-calendar"
+          class="su-calendar clickable icon default iconClickable"
+          role="button"
+          tabindex="0"
+        />
+      </div>
+      <input
+        placeholder="MM/DD/YYYY h:mm A"
+        type="text"
+        value=""
+      />
+    </label>
+  </div>
 </div>
 `;
 
+exports[`DatePicker should render date picker with time picker 2`] = `
+"<div>
+  <div class=\\"container\\">
+    <div style=\\"top: 10px; left: 34px; position: fixed; pointer-events: auto;\\">
+      <div class=\\"rdt\\">
+        <div></div>
+        <div class=\\"rdtPicker\\">
+          <div class=\\"rdtDays\\">
+            <table>
+              <thead>
+                <tr>
+                  <th class=\\"rdtPrev\\"><span>‹</span></th>
+                  <th class=\\"rdtSwitch\\" colspan=\\"5\\" data-value=\\"3\\">April 2017</th>
+                  <th class=\\"rdtNext\\"><span>
```

Comment:
Is that snapshot necessary? This test should only check if the `error` class is correctly appended to the previous snapshot. Also feels like this one is from somewhere before, because in this test the overlay is not even opened...